certbot-nginx/certbot_nginx/parser.py

```diff
@@ -216,8 +216,8 @@ def _parse_ssl_options(self, ssl_options):
                     return nginxparser.load(_file).spaced
             except IOError:
                 logger.warn("Missing NGINX TLS options file: %s", ssl_options)
-            except pyparsing.ParseBaseException as err:
-                logger.debug("Could not parse file: %s due to %s", ssl_options, err)
+            except pyparsing.ParseBaseException:
+                logger.debug("Could not parse file: %s", ssl_options)
```

Comment:
The changes here and in the other `except` block above look like a bad merge. I landed #4221 which changed this.